test/commands/forward.js

```diff
@@ -0,0 +1,35 @@
+var path = require('path');
+var child = require('child_process');
+
+function remove(arg) {
+  var port = arg.split(':')[1];
+  var command = "kill $(lsof -i :" + port + " | awk 'NR>1{ print $2; }')";
```

Comment:
Used double quotes as it contains embedded single quotes. No templates strings for the executable as it is all ES5 to save another transpilation.